Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml

```diff
@@ -96,7 +96,7 @@
     <string name="vaccination_card_status_vaccination_complete">"Gültig bis einschließlich %s"</string>
 
     <!-- XTXT: Vaccination QR code scan error message-->
-    <string name="error_vc_invalid">Dieser QR-Code ist kein gültiges Impfzertifikat.\n\nWeitere Informationen zum Erhalt Ihres Impfzertifikats finden Sie in den FAQ.</string>
+    <string name="error_vc_invalid">Dieser QR-Code ist kein gültiges Zertifikat.\n\nWeitere Informationen zum Erhalt Ihres Zertifikats finden Sie in den FAQ.</string>
```

Comment:
@NataliaLemmerth  Please let me know what is the right translation?

Reply:
@mtwalli It's correct. Thank you very much!